packages/middleware-endpoint/package.json

```
@@ -0,0 +1,56 @@
+{
+  "name": "@aws-sdk/middleware-endpoint",
+  "version": "3.130.0",
```

The version should be `3.0.0` to start with. Lerna will set it to appropriate version on publishing.
```suggestion
  "version": "3.0.0",
```